app/assets/javascripts/discourse/app/widgets/post-user-tip-shim.js

```diff
@@ -1,6 +1,8 @@
 import { hbs } from "ember-cli-htmlbars";
 import { registerWidgetShim } from "discourse/widgets/render-glimmer";
 
+// TODO deprecate for the Glimmer Post Menu
```

Comment:
What does this mean? We need to add it to the list of deprecated widgets in plugin-api.js?

Reply:
Exactly. I added the widget to the deprecated list